XPlat/DropboxApiV2/component/Copyright.txt

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2015 Dropbox Inc., http://www.dropbox.com/
```

Comment:
This file should be called `External-Dependency-Info.txt` and found in the root ( XPlat/DropboxApiV2)

If any of our NuGets or Components redistribute any of the Dropbox content we need to provide this file in that NuGet or Component.  The packaged file is to be called `THIRD-PARTY-NOTICES.txt`.

It seem that our NuGet may just depend on Dropbox's content.  If this is the case we do not need the file int he packages.

Reply:
Moved the file and added the standard 3rd party notices header

Reply:
no content is used